### PR TITLE
Changed moon cycle to be realistic.

### PR DESCRIPTION
**Pull request by arijust**
#### Summary

```SUMMARY: Bugfixes "Moon cycle taking 60 days."```
#### Purpose of change

Fixes moon. It isn't on issue list.
#### Describe the solution

Shorten moon cycle to 30 days.
#### Describe alternatives you've considered

Moon cycle stays as it is.
#### Testing

It shouldn't cause any errors. 
Well, probably.

### CODE REVIEW

File: src/calendar.cpp [hunk 1]
```diff
@@ -59,8 +59,8 @@ double default_daylight_level()
 
 moon_phase get_moon_phase( const time_point &p )
 {
-    //One full phase every 2 rl months = 2/3 season length
-    const time_duration moon_phase_duration = calendar::season_length() * 2.0 / 3.0;
+    //One full phase every 1 rl months = 1/3 season length
+    const time_duration moon_phase_duration = calendar::season_length() * 1.0 / 3.0;
```
Comment (Night-Pryanik) [inline src/calendar.cpp]:
```suggestion
    const time_duration moon_phase_duration = calendar::season_length() / 3.0;
```

**Comment by Qrox** [comment]
Looks like the vision test is failing due to the change to moon cycle.

**Comment by Mecares** [comment]
Why 30 days? Shouldnt it be something like 27 days considering it takes 27 days, 7 hours and 43,7 minutes in reality?

**Comment by arijust** [comment]
It seems my change, impaired vision range at night.
Setting this PR on WIP, until I can find reason for this bug.

**Comment by OzoneH3** [comment]
The problem is in vision_test.cpp itself, to get the correct full moon time you have to adjust it a bit.
```const time_point full_moon = calendar::turn_zero + calendar::season_length() / 3 / 2;```
Which will then point to the middle of a month which is the full_moon.

**Comment by arijust** [comment]
Ok thanks.

File: src/calendar.cpp [hunk 2]
```diff
@@ -63,6 +63,7 @@ moon_phase get_moon_phase( const time_point &p )
     const time_duration moon_phase_duration = calendar::season_length() / 3.0;
     //Switch moon phase at noon so it stays the same all night
     const time_duration current_day = ( p - calendar::turn_zero ) + 1_days / 2;
+    const time_point full_moon = calendar::turn_zero + calendar::season_length() / 3 / 2;
```
Comment (OzoneH3) [inline src/calendar.cpp]: Not here, in vision_test.cpp its in the tests folder.

Comment (arijust) [reply src/calendar.cpp]: Ok. Sorry.

**Pull request merged**